pkgs/servers/web-apps/wordpress/packages/wordpress-plugins.json

```diff
@@ -30,6 +30,7 @@
 , "wordpress-seo": "gpl3Only"
 , "worker": "gpl3Plus"
 , "wp-change-email-sender": "gpl2Plus"
+, "wp-fail2ban-addon-contact-form-7": "gpl3Plus"
```

Comment:
By the way, you *will* get a merge conflict on this line if either PR gets merged.

Reply:
There it is :innocent: